### ISSUE TITLE
ascii mode, paused construction 'P' is displayed over menus

### ISSUE THREAD

**Issue by fartron** [DFHack/dfhack#3560]
the green P character that is used for paused constructions when the display is in ascii is drawn over everything else. i don't use ascii much so i only noticed this by accident, might be true of other things drawn on the screen in that mode too

**Comment by myk002** [comment]
yeah, it might be an issue for all overlays. *maybe* we could fix this by registering a DF widget to better control the draw order. That's new territory, though, and I don't know if that will have the intended effect.

**Comment by fartron** [comment]
might be preferable to turn them off when a menu is detected then

**Comment by myk002** [comment]
It might be enough. There will be some side effects, like the overlays disappearing when the squads menu is open or even the build menu, but maybe this is ok or maybe we can work around it.
